packages/nimble-components/src/table/specs/table-keyboard-navigation-hld.md

```diff
@@ -0,0 +1,180 @@
+    - Possible confusion raised from how things like screen readers navigate the table, and how the table manages navigation.
+    - Implementation cost. If it is important to make the table work well with screen readers, and part of that involves when the screen reader has moved focus to a `gridcell` that has multiple interactive elements, we would need to reconcile what a user has to do to actually interact with either of those elements. This isn't necessarily free.
+    - While it's unclear if this would ever be a request, we would be prevented from allowing users to copy cell data strictly through the use of a keyboard (would require mouse highlighting).
+
+2. Pressing `Tab` either moves the focus _into_ or _out of_ the table. _Note: This was aligned with the ARIA `grid` pattern._
+
+    Pros:
+
+    - Gives users a fast means of tabbing past the table as part of a larger set of tabbable components.
+
+    Cons:
+
+    - Possible confusion for users when a row has focus, with obvious focusable/tabbable elements, and `Tab` does not take them to those elements.
+    - One could say that only adding the interactive elements in a row to the set of tabbable elements on a page isn't that much of an overhead from a navigation perspective, so trying to reduce it below that is an unnecessary goal.
+
+3. The row selection checkbox is not part of the navigable set of elements on a row.
+
+    Pros:
+
+    - Avoids an interactive redundancy in that selecting a row is expected to be done by focusing the row and pressing `<Space>`.
+    - Provides consistency with the `GroupRow` where we don't allow a user to move focus to the checkbox, though this is only a pro on its own if ARIA properly informed the user that the GroupRow could be operated as if it was the checkbox.
+
+    Cons:
+
+    - Breaks ARIA guidance that all interactive elements be focusable.
+    - Behavior that doesn't match mouse interactions may cause confusion or frustration that interrupts users' goals/workflows.
+
+4. Expand/collapse of rows uses `Alt-ArrowUp/ArrowDown` (instead of simply `ArrowLeft` and `ArrowRight`)
+
+    Cons:
+
+    - Reliance on modifier keys when there are available workflows/keys that do not require them (i.e. strictly using `ArrowLeft` and `ArrowRight` keys with no modifiers)
+    - Mistypes (attmpted to use `Alt` modifier but didn't) shifts focus unexpectedly.
+
+## Open Issues
+
+-   Row selection: Should pressing `Space` when a single cell is focused, result in selecting that row?
+-   When focus leaves the table and later returns, should we return focus to the previously focused row, or restart focus on the top row/ header?
```

Comment:
A point of reference is that the AzDO table (at least the one for the backlog) retains what row should be focused when the table receives focus again. I would be fine with this behavior.

Reply:
Removed from open issues. Kept the wording that we'll return focus to the previously focused row.